devenv.nix

```diff
@@ -141,35 +143,41 @@ let
     TARGETFOLDER="${config.env.DEVENV_STATE}/importdb"
 
     rm -rf "$TARGETFOLDER"
-
     set -e
 
     if [[ "$1" == *.sql ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.gz ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.gz"
-        gunzip -c "$TARGETFOLDER/latest.sql.gz" > "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.zip ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.zip"
-        unzip -j -o "$TARGETFOLDER/latest.sql.zip" '*.sql' -d "$TARGETFOLDER"
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
+    elif [[ "$1" == *.gz ]]; then
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.gz"
+      ${pkgs.gzip}/bin/gunzip -c "$TARGETFOLDER/latest.sql.gz" > "$TARGETFOLDER/dump.sql"
+    elif [[ "$1" == *.zip ]]; then
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.zip"
+
+      if [ x"$IMPORT_DATABASE_PASSWORD" == "x" ]; then
+        UNZIP_COMMAND="${pkgs.unzip}/bin/unzip"
+      else
+        UNZIP_COMMAND="${pkgs.unzip}/bin/unzip -P $IMPORT_DATABASE_PASSWORD"
+      fi
+
+      $UNZIP_COMMAND -j -o "$TARGETFOLDER/latest.sql.zip" '*.sql' -d "$TARGETFOLDER"
     else
-        echo "Unsupported file type for file at $1"
-        exit
+      echo "Unsupported file type for file at $1"
+      exit
     fi
 
     rm -f "$TARGETFOLDER/latest.sql.*"
 
     SQL_FILE=$(find "$TARGETFOLDER" -name "*.sql" | head -n 1)
 
     if [[ "$SQL_FILE" == "" ]]; then
-        echo "No SQL file found"
-        exit
+      echo "No SQL file found"
+      exit
     fi
 
-    LANG=C LC_CTYPE=C LC_ALL=C sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' "$SQL_FILE"
-    LANG=C LC_CTYPE=C LC_ALL=C sed 's/NO_AUTO_CREATE_USER//' "$SQL_FILE"
+    LANG=C LC_CTYPE=C LC_ALL=C ${pkgs.gnused}/bin/sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' "$SQL_FILE" > $SQL_FILE
+    LANG=C LC_CTYPE=C LC_ALL=C ${pkgs.gnused}/bin/sed 's/NO_AUTO_CREATE_USER//' "$SQL_FILE" > $SQL_FILE
 
-    MYSQL_PWD="" ${config.services.mysql.package}/bin/mysql shopware -f < "$SQL_FILE"
+    MYSQL_PWD="" ${config.services.mysql.package}/bin/mysql -u root shopware -f < "$SQL_FILE"
```

Comment:
Our devenv configuration sets the username for mysql to `shopware`. Why would we need `root` here?

Reply:
the package does not use the provided configurations. Its fixed now.